### PR TITLE
Change the default channel in `dotnet-install` scripts to "preview".

### PR DESCRIPTION
**Pull request by eerhardt**
@krwq @piotrpMSFT

### CODE REVIEW

File: scripts/obtain/dotnet-install.ps1 [hunk 1]
```diff
@@ -47,7 +47,7 @@
 #>
 [cmdletbinding()]
 param(
-   [string]$Channel="beta",
```
Comment (piotrpMSFT) [inline scripts/obtain/dotnet-install.ps1]: Can you also update install.[ps1|sh]?

Comment (eerhardt) [inline scripts/obtain/dotnet-install.ps1]: oh right, they are duped still?

How about we delete the "bad" one?

**Comment by livarcocc** [comment]
:shipit:

File: scripts/obtain/dotnet-install.ps1 (hunk 1 above)
Comment (eerhardt) [inline scripts/obtain/dotnet-install.ps1]: @piotrpMSFT @krwq - if `dotnet-install` is the "real one" going forward, I'm gonna merge this, and then make a new PR to remove the old one.

Everyone cool with that?

**Pull request merged**